assets/css/style.css

```diff
@@ -0,0 +1,223 @@
+body {
+    background-color: white;
+    margin: 0;
+    font-family: Arial, Helvetica, sans-serif
+}
+
+ul {
+    list-style-type: none;
+    overflow: hidden;
+    margin: 0;
+    padding: 0;
+    grid-column: 4/6;
+    grid-row: 1/2;
+    text-align: center;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-column-gap: 1px;
+    background-color: rgb(26, 25, 25)
+}
+
+li {
+    text-align: center;
+}
+
+li a {
+    display: block;
+    text-decoration: none;
+    text-align: center;
+    padding: 14px 16px;
+    color: whitesmoke;
+}
+
+.container {
+    display: grid;
+    grid-template-columns: 1fr 120px 2fr 2fr 1fr;
+    grid-column-gap: 0;
+    grid-row-gap: 15px;
+    margin: 0;
+}
+
+.container2 {
+    grid-column: 2/5;
+    grid-row: 3/4;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 15px;
+}
+
+#mearat {
+    grid-column: 2/3;
+    grid-row: 1/2;
+    background-color: rgb(63, 62, 62);
+    color: whitesmoke;
+    padding-top: 15px;
+    margin-bottom: 2px solid rgb(36, 92, 156);
+    text-decoration: none;
+    text-align: center;
+    font-weight: bold;
+    border-bottom: 4px solid grey;
+}
+
+#home {
+    text-decoration: none;
+    background-color: rgb(63, 62, 62);
+    color: whitesmoke;
+    padding-top: 20px;
+}
+
+.aboutme {
+    grid-column: 2/5;
+    grid-row: 2/3;
+    text-align: left;
+    font-size: 20px;
+}
+
+.content {
+    grid-column: 2/5;
+    grid-row: 4/5;
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.hangman {
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    font-size: 60px;
+    align-content: center;
+    text-align: center;
+    padding: 15px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.rpggame {
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    font-size: 60px;
+    align-content: center;
+    text-align: center;
+    padding: 15px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.triviagame {
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    font-size: 60px;
+    align-content: center;
+    text-align: center;
+    padding: 15px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
this box shadow is repeated multiple places, in cases like that, move them onto their own class.
```css
.shadow {
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
}
```